dockerfiles/Dockerfile_full

```diff
@@ -1,25 +1,25 @@
 FROM heroku/heroku:18
 
 ENV DEBIAN_FRONTEND=noninteractive
+ARG SALESFORCE_CLI_VERSION=latest
 RUN apt-get update
 RUN echo '2d316e55994086e41761b0c657e0027e9d16d7160d3f8854cc9dc7615b99a526  ./nodejs.tar.gz' > node-file-lock.sha \
   && curl -s -o nodejs.tar.gz https://nodejs.org/dist/v12.18.2/node-v12.18.2-linux-x64.tar.gz \
   && shasum --check node-file-lock.sha
 RUN mkdir /usr/local/lib/nodejs \
   && tar xf nodejs.tar.gz -C /usr/local/lib/nodejs/ --strip-components 1 \
   && rm nodejs.tar.gz node-file-lock.sha
+ENV PATH=/usr/local/lib/nodejs/bin:$PATH
+RUN npm install --global sfdx-cli@${SALESFORCE_CLI_VERSION}
```

Comment:
In a way it is really unsatisfying that this works. I am certain I tried this solution and may more before I talked you the team abut the permission issues I was having.

For some reason it just started working when I was showing Prakash when I tried.